deepmd/fit/ener.py

```diff
@@ -549,16 +554,20 @@ def build(
             assert len(bias_atom_e) == self.ntypes
 
         fparam = None
-        aparam = None
         if self.numb_fparam > 0:
             fparam = input_dict["fparam"]
             fparam = tf.reshape(fparam, [-1, self.numb_fparam])
             fparam = (fparam - t_fparam_avg) * t_fparam_istd
-        if self.numb_aparam > 0:
-            aparam = input_dict["aparam"]
-            aparam = tf.reshape(aparam, [-1, self.numb_aparam])
-            aparam = (aparam - t_aparam_avg) * t_aparam_istd
-            aparam = tf.reshape(aparam, [-1, self.numb_aparam * natoms[0]])
+
+        if self.use_aparam_as_mask:
+            aparam = None
+        else:
+            aparam = None
```

Comment:
It seems that `aparam = None` can be put outside the `if...else`.

Reply:
Fixed in latest commit.

Reply:
Fixed in latest commit.